package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "edx-ora2",
-  "version": "2.8.1",
+  "version": "2.8.3",
```

Comment:
Is there a matching change in `setup.py` and `package-lock.json`?

Reply:
Probably. I need to see if a merge was fubar'd.